### ISSUE TITLE
Feature Request: Rename with reference to file property

### ISSUE THREAD

**Issue by Bleakwar** [chrdavis/SmartRename#31]
On music files I tend to find myself Needing to rename a file to match the "Title" property because for whatever reason lot of programs are incapable of renaming the file name and only fill out the file properties. I've attached a photo of what I'm talking about.
![Screenshot_20230106_113303](https://user-images.githubusercontent.com/59183820/211056771-71425af6-a83d-4c2e-96f7-4065497f6334.png)

**Comment by Wolterhon** [comment]
Yep! I just had to rename a bunch of music files, their names contained just names of tracks, and I'd like then to have artists' names too. I thought it was possible to do with SmartRename but it's sadly not.

**Comment by sjsilverman** [comment]
Apologize in advance if this isn't appropriate for github repository (let me know, I'm new here).  You may be interested in media monkey www.mediamonkey.com which has this capability.  I haven't used it in a while (since switching from mp3s to streaming), but it used to be very good, and quick look at the page seems to show it is still actively in development and supported. (there is a free version, but it's not open source.)
